com.CalculaPro.app/com.CalculaPro.app.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "com.CalculaPro.app",
```

Comment:
`app` is not a proper application name. Sould be something like `CalculaPro`

Reply:
The domain part should always be in lowercase. This is written in the ID requirements that was ticked